cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -160,3 +160,159 @@ def test_vmware_vds_ui_tagging():
             4.You can assign tags to DSwitch.
     """
     pass
+
+
+@pytest.mark.manual()
+@pytest.mark.tier(1)
+def test_vmware_inaccessible_datastore():
+    """
+    VMware sometimes has datastores that are inaccessible, and CloudForms should indicate that.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1684656
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testtype: integration
+        testSteps:
+            1.Integrate VMware provider in CFME
+            2.Compute > Infrastructure > Datastores
+            3.Check if any of the datastores marked inaccessible and compare it with VMware UI.
+        expectedResults:
+            1.
+            2.Datastores page opens showing all the datastores known to CFME
+            3.All datastores that are inaccessible in vSphere should be marked such in CFME UI too.
+    """
+    pass
+
+
+@pytest.mark.manual()
+@pytest.mark.tier(1)
+def test_vmware_cdrom_dropdown_not_blank():
+    """
+    Test CD/DVD Drives dropdown lists ISO files, dropdown is not blank
+    https://bugzilla.redhat.com/show_bug.cgi?id=1689369
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testtype: integration
+        testSteps:
+            1.Integrate VMware provider in CFME
+            2.Compute > Infrastructure > Datastores
+            3.Run SSA on datastore which contains ISO files
+            4.Navigate to Compute>Infrastructure>Virtual Machines, select any virtual machine
+            5.Reconfigure it to have new ISO file attached to it in CD/DVD drive
+        expectedResults:
+            1.
+            2.Datastores page opens showing all the datastores known to CFME
+            3.SSA runs successfully, and you can see files in datastore
+            4.Virtual machine is selected
+            5.Dropdown of ISO files is not empty for CD/DVD Drive
+    """
+    pass
+
+
+@pytest.mark.manual()
+@pytest.mark.tier(1)
+def test_vmware_publish_to_template():
```

Comment:
This seems to be automated for RHEV
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/infrastructure/test_publish_vm_to_template.py#L39
May be you can check with @lina-nikiforova if it can be automated for VMware too in same test

Reply:
Opening new PR to add my tests within same file where Angelina has a RHV Test.

Reply:
PR #8692